pkg/k8s/watchers/pod.go

```diff
@@ -396,6 +396,11 @@ func updateCiliumEndpointLabels(ep *endpoint.Endpoint, labels map[string]string)
 		controller.ControllerParams{
 			DoFunc: func(ctx context.Context) (err error) {
 				pod := ep.GetPod()
+				if pod == nil {
+					err := errors.New("Skipping CiliumEndpoint update because it has no k8s pod")
```

Comment:
Can we add some extra labels to the log message to give context regarding what endpoint this is referring to? (i.e. include the endpoint ID such as here: https://github.com/cilium/cilium/blob/master/pkg/k8s/watchers/pod.go#L426)

Reply:
@tommyp1ckles just a FYI, when a controller fails, it will show up under `cilium status`. The name of this controller already contains the ep ID defined on line 389.